Adds info about console.table()

edwardball opened the pull request:
Hi, I've added a line about console.table() to the info about log/warn/error etc. This is my first contrib to open source using Git so I hope I've done everything right, let me know if not!

Thanks

Commit:
Adds info about console.table()

andismith commented:
Hi Aptary,

Thanks for submitting! This is a really great tip. I think it may be best to create a new tip for console.table since it displays the information in such a different way to the other logging items.

What do you think?

Andy

Commit:
Adds brand new page for console.table()

edwardball commented:
Hi Andy, 

That sounds like a good idea. I've removed my changes to the log/warn/error page and added a new page for console.table, with a couple of images too.

Aptary

Commit:
Fixes typo on line 24 of secret/console-table.md

andismith commented:
Brilliant! Thank you!
